python/pyspark/pandas/frame.py

```diff
@@ -10081,6 +10086,10 @@ def all(self, axis: Axis = 0) -> "Series":
             * 0 / 'index' : reduce the index, return a Series whose index is the
               original column labels.
 
+        bool_only : bool, default None
+            Include only boolean columns. If None, will attempt to use everything,
+            then use only boolean data.
```

Comment:
Don't we need `Not implemented for Series.` ??

Seems like it's mentioned in pandas documents as below:
<img width="641" alt="Screen Shot 2022-03-29 at 7 42 37 AM" src="https://user-images.githubusercontent.com/44108233/160498960-000043f6-5ecd-4652-a143-6b541a5fd331.png">

Reply:
We may not. In pandas, Series and DataFrame share the same docstring. Ours are separate.

Reply:
Let me know if you still think it necessary.

Reply:
Got it. Seems fine as is.